bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/ide/dialogs/ResourceFilterGroup.java

```diff
@@ -2772,8 +2772,8 @@ public void widgetSelected(SelectionEvent e) {
 		setupDescriptionText(null);
 	}
 
-	private String[] timeIntervalPrefixes = {"s", "m", "h", "d"};    //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$ //$NON-NLS-4$
-	private double[] timeIntervalScale = {60, 60, 24};
+	private static final String[] timeIntervalPrefixes = { "s", "m", "h", "d" }; //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$ //$NON-NLS-4$
```

Comment:
Making it static final is not related to the pr topic.  That’s okay, but generally static final fields use upper case names in the existing code base. I think we had that discussion…